index.html

```diff
@@ -127,13 +127,13 @@ <h2>Automating tasks for fun and profit</h2>
 
 <pre>
 <span class="comment"># List issues assigned to you that are labeled "urgent"</span>
-<span class=p></span><strong>hub issue</strong> -a YOUR_USER -l urgent
+<span class=p></span><strong>hub issue</strong> --assignee YOUR_USER --labels urgent
 
 <span class="comment"># List the URLs of at most 20 pull requests based on the "develop" branch:</span>
-<span class=p></span><strong>hub pr list</strong> -L 20 -b develop --format='%t [%H] | %U%n'
+<span class=p></span><strong>hub pr list</strong> --limit 20 --base develop --format='%t [%H] | %U%n'
 
 <span class="comment"># Create a GitHub Release from master using release notes from a file</span>
-<span class=p></span><strong>hub release create</strong> -c -F release-notes.txt v2.3.0
+<span class=p></span><strong>hub release create</strong> --copy --file release-notes.txt v2.3.0
```

Comment:
Since a filename immediately follows `-F`, I wonder whether that's enough for the reader to understand that this is the flag for passing a file and that we don't need to expand it. I'm reading through these changed examples now and thinking that we could maybe get away with shortening `--file` back into `-F`

Reply:
I agree, fixed.